### PR TITLE
Loitering docs

### PR DESCRIPTION
**Pull request by hanryxu**
## What does this PR include?
Add docs and demo Jupyter notebook to loitering example.

## User API changes
None

## How did I test the PR?
Jupyter notebook tested in Google Colab

### CODE REVIEW

File: examples/loitering/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Detect people loitering example
+
+## Setup
+
+You can follow the instructions [here](../../vqpy/README.md) to prepare your environment for VQPy. No other dependency is used.
+
+Video used in this query can be found [here](https://youtu.be/EuLMrUFNRxQ).
```
Comment (yushan111) [inline examples/loitering/README.md:7]: mention that video need to be downloaded to local

File: examples/loitering/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Query logic
```
Comment (yushan111) [inline examples/loitering/README.md:9]: Put this part in colab notebook

File: examples/loitering/README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    ...
+    "in_roi_periods": None,
+}
+```
+
+## Run example
+
+To run locally
+
+```python
```
Comment (yushan111) [inline examples/loitering/README.md:62]: bash

File: examples/loitering/README.md (hunk 2 above)
Comment (hanryxu) [reply examples/loitering/README.md:9]: Was mentioned in the notebook.

File: examples/loitering/README.md (hunk 1 above)
Comment (hanryxu) [reply examples/loitering/README.md:7]: Added instruction to section "Run example"

File: examples/loitering/README.md (hunk 3 above)
Comment (hanryxu) [reply examples/loitering/README.md]: Corrected

File: examples/loitering/README.md (hunk 2 above)
Comment (hanryxu) [reply examples/loitering/README.md]: Moved

**Review by yushan111 — APPROVED** [review]
LGTM

File: examples/loitering/demo.ipynb [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+   "source": [
+    "#### Selecting VObjs' properties for output"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For each person, we can output its tracking id, coordinate, and duration of staying in the region:\n",
```
Comment (yushan111) [inline examples/loitering/demo.ipynb:200]: For each Person VObj that satisfies the filter conditions

File: examples/loitering/demo.ipynb [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+    "\n",
+    "- tracking id can be selected with `track_id`\n",
+    "- outputting coordinate needs to select `coordinate` then serialize the ndarray to string with post-processing function `lambda x: str(x)`\n",
```
Comment (yushan111) [inline examples/loitering/demo.ipynb:203]: 1. In select cons, the key is a property name, which can be either a property in VObj, or a property in our built-in property library. The value is a post-processing function you want to apply to the property as an output. 
2. maybe explain a little more about the difference between `coordinate` and the `bottom_center` you used in filtering, i.e why you use `bottom_center` in filter and `coordinate` as output.
3. And maybe explain more where the names come from: in vobj/in property lib/introduced by continuing ..  for track_id and coordinate

File: examples/loitering/demo.ipynb [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+    "    \"in_roi_periods\": None,\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The query"
```
Comment (yushan111) [inline examples/loitering/demo.ipynb:236]: Construct the query with `filter_cons` and `select_cons`

File: examples/loitering/demo.ipynb [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+    "save_folder = \"./vqpy_outputs\"\n",
+    "detector_model_dir = \"./\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```
Comment (yushan111) [inline examples/loitering/demo.ipynb:285]: Next, we can run the query by `vqpy.launch`. 
For `cls_name`, ...
For `cls_type`, ...

File: examples/loitering/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- `--path`: path of video;
+- `--save_folder`: the folder to save query result;
+- `-d`: directory containing pre-trained models (only model for YOLOX is used).
+
+### Run in Jupyter notebook
+
+Or run [demo.ipynb](./demo.ipynb). The notebook also contains more details about the query.
+
+Notebook is tested in Google Colab, it's advised to use a unused Python3.8 environment if you prefer to run it locally.
```
[the remark this answers is not in the document]
Comment (hanryxu) [reply examples/loitering/README.md:28]: Google colab has Python 3.8 by default. Also mentioned Python 3.8 should be used in demo.ipynb.

File: examples/loitering/demo.ipynb [hunk 9]
```diff
@@ -0,0 +1,356 @@
...
+    "## Environment setup"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please download video from [here](https://youtu.be/EuLMrUFNRxQ) and place it in the same directory as this notebook, aside from running the cell below."
```
[the remark this answers is not in the document]
Comment (hanryxu) [reply examples/loitering/demo.ipynb:33]: Added example working directory structure.

File: examples/loitering/demo.ipynb (hunk 5 above)
Comment (hanryxu) [reply examples/loitering/demo.ipynb]: Added more explanation

**Comment by yushan111** [comment]
Move step 3 to "Running the query"
Others LGTM

**Pull request merged**